quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -135,13 +135,36 @@ impl Actor for ControlPlane {
             .context("failed to initialize ingest controller")?;
 
         self.handle(RefreshPlanLoop, ctx).await?;
+
         ctx.schedule_self_msg(CONTROL_PLAN_LOOP_INTERVAL, ControlPlanLoop)
             .await;
 
         Ok(())
     }
 }
 
+#[async_trait]
+impl Handler<RefreshPlanLoop> for ControlPlane {
```

Comment:
I think we can remove it. (I assume it has resurfaced during the rebase?)

Reply:
I'm one rebase behind.